src/components/Login/Login.js

```diff
@@ -11,6 +14,8 @@ import "./Login.scss";
 const Login = () => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [authError, setAuthError] = useState("");
+  const nav = useNavigate();
```

Comment:
I believe `navigate` is more expressive than `nav` since the nav can be the nav bar at the top, while navigate indicates an action.
```suggestion
  const navigate = useNavigate();
```